tracer/contrib/database/sqltraced/mysql/collations.go

```diff
@@ -0,0 +1,25 @@
+// Go MySQL Driver - A MySQL-Driver for Go's database/sql package
```

Comment:
read up on the [vendor](https://blog.gopheracademy.com/advent-2015/vendor-folder/) folder. I imagine most of this stuff should go there